bin/nanoid.test.js

```diff
@@ -1,11 +1,15 @@
+let { test } = require('uvu')
+let assert = require('uvu/assert')
```

Comment:
In my other open source projects I use `let { equal, is } = require()` style. Can you convert it too?